ortools/linear_solver/xpress_interface.cc

```diff
@@ -223,8 +223,8 @@ class XpressInterface : public MPSolverInterface {
     sync_status_ = MUST_RELOAD;
   }
   // Adds a new feasible, infeasible or partial MIP solution for the problem to the Optimizer.
-  // The hint is read in the MPSolver where the user set it
-  virtual void SetHint();
+  // The hint is read in the MPSolver where the user set it using SetHint()
+  void AddSolutionHintToOptimizer();
```

Comment:
Since you don't really need to interact with `XpressInterface`, you could use a free function such as
```cpp
void AddSolutionHintToOptimizer(const XPRSprob& mLp)
```

Reply:
I do need this->solver_ to access solver_->solution_hint_